backend/app/core/schedulers/ap_scheduler.py

```diff
@@ -55,7 +54,7 @@ def shutdown(self):
     def add_schedule(self, schedule: Schedule, datasource_id: str, dataset_id: str):
         return self.ap_scheduler.add_job(
             id=f"{datasource_id}__{dataset_id}__{uuid.uuid4()}",
-            func=run_dataset_validation,
+            func=run_validation.delay,
```

Comment:
That's game changer 👏 The scheduler is now a super light process that just... schedule things